plugins/modules/vmware_dvs_portgroup_info.py

```diff
@@ -184,19 +197,51 @@ def gather_dvs_portgroup_info(self):
         dvs_lists = self.dvsls
         result = dict()
         for dvs in dvs_lists:
+            switch_supports_mac_learning = self.supports_mac_learning(dvs)
             result[dvs.name] = list()
             for dvs_pg in dvs.portgroup:
+                mac_learning = dict()
                 network_policy = dict()
                 teaming_policy = dict()
                 port_policy = dict()
                 vlan_info = dict()
+                active_uplinks = list()
+                standby_uplinks = list()
+
+                if dvs_pg.config.type == 'ephemeral':
+                    port_binding = 'ephemeral'
+                else:
+                    port_binding = 'static'
+
+                if dvs_pg.config.autoExpand is True:
+                    port_allocation = 'elastic'
+                else:
+                    port_allocation = 'fixed'
+
+                # If the dvSwitch supports MAC learning, it's a version where securityPolicy is deprecated
+                if self.module.params['show_network_policy']:
+                    if switch_supports_mac_learning and dvs_pg.config.defaultPortConfig.macManagementPolicy:
```

Comment:
I'm not sure because I don't have VCSA 6.5 or less, according to the API documentation, I seem that the `DVSMacManagementPolicy` doesn't support VCSA 6.5 or less.  
Maybe the module doesn't work fine on the 6.5 or less.  
VCSA 6.0 is the end of general support but, 6.5 is supported by VMware yet.  
https://lifecycle.vmware.com/#/

I think that I need a note about it to the module.  
What do you think?

Reply:
> I'm not sure because I don't have VCSA 6.5 or less, according to the API documentation, I seem that the `DVSMacManagementPolicy` doesn't support VCSA 6.5 or less.
> Maybe the module doesn't work fine on the 6.5 or less.

When people are still running an older version that doesn't support `DVSMacManagementPolicy`, I fall back to the original / old code:

https://github.com/mariolenz/community.vmware/blob/b800d291a58c1fc51be24ceea4377931d4fccdcb/plugins/modules/vmware_dvs_portgroup_info.py#L231

Does this answer your question?

Reply:
> Does this answer your question?

yes, thanks, @mariolenz :)